post-processor/vagrant-cloud/client.go

```diff
@@ -26,14 +26,20 @@ type VagrantCloudClient struct {
 }
 
 type VagrantCloudErrors struct {
-	Errors map[string][]string `json:"errors"`
+	Errors []interface{} `json:"errors"`
```

Comment:
What do you think about typing this error slice ?
```suggestion
	Errors []error `json:"errors"`
```

They could be created with the errors pkg: https://golang.org/pkg/errors/#New

Reply:
( this would make for an idiomatic error slice :D )

Reply:
Oops sorry I didn't realise this was being deserialised ! My suggestion will not work !

Reply:
I like that idea. Would I have to add a new field that can be used for unmarshalling the response?

Reply:
:+1: